src/app/pages/ledger/connect-ledger/connect-ledger.layout.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Stack } from '@stacks/ui';
+
+import { Divider } from '@app/components/divider';
+import { BaseDrawer } from '@app/components/drawer';
+import { Link } from '@app/components/link';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Caption } from '@app/components/typography';
+import ConnectLedger from '@assets/images/onboarding/ledger/connect-ledger.png';
+// import { WarningLabel } from '@app/components/warning-label';
+
+import { LookingForLedger } from './components/looking-for-ledger';
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface ConnectLedgerLayoutProps {
+  isLookingForLedger: boolean;
+  onCancelConnectLedger(): void;
+  onConnectLedger(): void;
+}
+export function ConnectLedgerLayout(props: ConnectLedgerLayoutProps) {
+  const { isLookingForLedger, onCancelConnectLedger, onConnectLedger } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Stack spacing="extra-loose" width="331px">
+          {/* TODO: Implement warning with actual ledger integration */}
+          {/* <WarningLabel>Before proceeding, close Ledger Live if it’s open</WarningLabel> */}
+          <Box mt="tight">
+            <img src={ConnectLedger} />
+          </Box>
+        </Stack>
```

Comment:
I think this is another example of maybe using stack a little bit too much. There's only 1 child, so what is it spacing?

Reply:
The `extra-loose` spacing here just wasn't enough (32px) so I added the extra to `mt` to use the variables from the UI library. I can remove the variables here and just use "40px" for the spacing and it will be the same.

When I started at Hiro I worked on the explorer and this pattern of nesting Stacks is heavily used in that code, so I thought it was how we were supposed to work with our UI library. I will stop nesting it.

Reply:
I'm sure there are some cases for it being nested, so wouldn't rule it out completely.

>  I can remove the variables here and just use "40px" for the spacing and it will be the same.

This reckon this is better. Otherwise when inspecting in dev tools, you see the margin applied in two places, which is confusing.

If it seems like I have an anti-stack agenda, I don't lol, it's a great component with many uses, just we should use it a little more sparingly, when components have exact spacing.

Reply:
Yep, got it, thanks for the feedback. I have refactored many of the Stacks > Flex now. I get that it is better in this case. Going to resolve this conversation.

Reply:
Our UI library is far from perfect. Moving forward we can work with @landitus to define a set of spaces that we both use in figma and the implementation alike. That way no design would stretch the limits to what we can implement with the system and we keep better consistency across the application.

Fernando also knows his way around CSS, and he has an appreciation for systems that create a bridge from design to code. I have spent countless hours in our past lives discussing how many spaces a system needs, are too many.

Reply:
We'll make it work for sure! thanks for pointing these issues out @fbwoolf